src/main/java/MainWindow.java

```diff
@@ -0,0 +1,76 @@
+import duke.DialogBox;
+import duke.Duke;
+
+import duke.MainUi;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Duke duke;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
+    private Image dukeImage = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
+
+    /**
+     * Initialise MainWindow.
+     */
+    @FXML
+    public void initialize() {
+        scrollPane.vvalueProperty().bind(dialogContainer.heightProperty());
+        MainUi tmpUi = new MainUi();
+        tmpUi.showWelcome();
+        dialogContainer.getChildren()
+            .add(DialogBox.getDukeDialog(new Label(tmpUi.getResponse()), new ImageView(dukeImage)));
+    }
+
+    public void setDuke(Duke d) {
```

Comment:
Write a javadoc to explain this method.

Reply:
IMO, this is self-explanatory and simple. Sometimes it's hard for me to justify whether getters & setters deserve JavaDoc.